app/code/Magento/PageCache/Test/Mftf/ActionGroup/AssertValueIsNotCachedActionGroup.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertValueIsNotCachedActionGroup">
+        <annotations>
+            <description>Assert that cached value is not the same as updated.</description>
```

Comment:
Please describe it more. Where this assert performed, on which page, on storefront or admin?